lib/ansible/module_utils/basic.py

```diff
@@ -1983,29 +1983,15 @@ def atomic_move(self, src, dest, unsafe_writes=False):
                             e = get_exception()
                             if e.errno != errno.EPERM:
                                 raise
-                        os.rename(b_tmp_dest_name, b_dest)
+                        try:
```

Comment:
so this changes the point of coverage of unsafe_writes, it will not be used as fallback except for the narrower error on temp file renaming.